### PR TITLE
Replace Jquery -> jQuery

### PR DESCRIPTION
**Pull request by lipis**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [ ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by lipis** [comment]
The occurrences were so few that I think it's easier to review in on PR..

**Review by lawrencejon — APPROVED** [review]
@lipis Good work!

File: guide/arabic/jquery/jquery-html-method/index.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
----
# طريقة HTML
-
+---
+title: HTML Method
+localeTitle: طريقة HTML
+---
+# طريقة HTML
+jQuery
 تحصل طريقة Jquery `.html()` على محتوى عنصر HTML أو تقوم بتعيين محتوى عنصر HTML.
```
Comment (raisedadead) [inline guide/arabic/jquery/jquery-html-method/index.md]: Is this an intended change? Doesn't look right here?

File: guide/chinese/jquery/jquery-html-method/index.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
----
# HTML方法
-
+---
+title: HTML Method
+localeTitle: HTML方法
+---
+# HTML方法
+jQuery
 Jquery `.html()`方法获取HTML元素的内容或设置HTML元素的内容。
```
Comment (raisedadead) [inline guide/chinese/jquery/jquery-html-method/index.md]: Is this an intended change? Doesn't look right here?

File: guide/spanish/jquery/jquery-html-method/index.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
----
# Método HTML
-
+---
+title: HTML Method
+localeTitle: Método HTML
+---
+# Método HTML
+jQuery
 El método `.html()` Jquery obtiene el contenido de un elemento HTML o establece el contenido de un elemento HTML.
```
Comment (raisedadead) [inline guide/spanish/jquery/jquery-html-method/index.md]: Is this an intended change? Doesn't look right here?

**Review by raisedadead — CHANGES_REQUESTED** [review]
The changes in the language file dont look right.

File: guide/arabic/jquery/jquery-html-method/index.md (hunk 1 above)
Comment (lipis) [reply guide/arabic/jquery/jquery-html-method/index.md]: `^M` what was that?

Comment (raisedadead) [reply guide/arabic/jquery/jquery-html-method/index.md]: I am not sure, however currently you have:

```
jQuery
تحصل طريقة Jquery `.html()` على محتوى عنصر HTML أو تقوم بتعيين محتوى عنصر HTML.
```

and similar changes in other files.

Comment (lipis) [reply guide/arabic/jquery/jquery-html-method/index.md]: not anymore

**Comment by raisedadead** [comment]
Okay, I see you have taken care of 

```
jQuery
تحصل طريقة Jquery `.html()` على محتوى عنصر HTML أو تقوم بتعيين محتوى عنصر HTML.
```

by removing the `jQuery` from each language.

but,

```
تحصل طريقة Jquery `.html()` على محتوى عنصر HTML أو تقوم بتعيين محتوى عنصر HTML.
```

Still needs fixing for the `Jquery` ?

**Comment by lipis** [comment]
@raisedadead You're right.. I thought it was part of the code or something.. there are no occurrences of `Jquery` now.

File: guide/russian/jquery/jquery-html-method/index.md [hunk 4]
```diff
@@ -1,8 +1,9 @@
...
-localeTitle: Метод HTML
----
# Метод HTML
-
+---
+title: HTML Method
+localeTitle: Метод HTML
+---
+# Метод HTML
+jQuery
```
Comment (raisedadead) [inline guide/russian/jquery/jquery-html-method/index.md]:
```suggestion

```

File: guide/russian/jquery/jquery-html-method/index.md [hunk 5]
```diff
@@ -1,8 +1,9 @@
...
 Метод JQuery `.html()` получает содержимое элемента HTML или задает содержимое элемента HTML.
```
Comment (raisedadead) [inline guide/russian/jquery/jquery-html-method/index.md]:
```suggestion
Метод jQuery `.html()` получает содержимое элемента HTML или задает содержимое элемента HTML.
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
@lipis I found one file in the wild!

I have left them as suggestions, so you can simply hit the apply suggestion button.

Thanks a lot for working on this.

**Comment by lipis** [comment]
Should be OK now.. (last famous words 😄)

**Review by raisedadead — APPROVED** [review]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @lipis 

Happy Contributing!

**Comment by raisedadead** [comment]
Hey @RandellDawson Only one PR affects this, and is in discussion anyways. I think we are good to merge this. If you can confirm, please go ahead and hit the merge button.

**Comment by RandellDawson** [comment]
I can confirm there is only one PR outstanding which is in discussion and this PR now looks good.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**